source/tutorials/profiling-gpu/profiling-gpu.rst

```diff
@@ -0,0 +1,135 @@
+Profiling GPU
+########################################################################
+
+HW pre-requisites
+===========================
+Build machine/PC with Ubuntu 16.04
+
+USB drive 8GB, ext4 formatted.
+
+USB Type-A to Type-A cable for build machine to target connection.
```

Comment:
Add comment that this USB cable is used only adb is enabled on the specific USB port